Ricochet/Assets/_Scripts/Player/PlayerController.cs

```diff
@@ -373,6 +381,12 @@ private void Move()
             else
             {
                 // flying
+                if (gameManagerInstance != null || GameManager.TryGetInstance(out gameManagerInstance))
+                {
+                    //TODO come back to this with a coroutine?
```

Comment:
I'm honestly okay with there not being a flying sound since players are going to be flying like 95% of the time and there are up to 4 of them at once.